cmd/tailscale/cli/web.go

```diff
@@ -412,6 +425,8 @@ func webHandler(w http.ResponseWriter, r *http.Request) {
 		LicensesURL:  licensesURL(),
 		TUNMode:      st.TUN,
 		IsSynology:   distro.Get() == distro.Synology || envknob.Bool("TS_FAKE_SYNOLOGY"),
+		DSMVersion:   distro.DSMVersion(), // 6 or 7, if IsSynology=true
```

Comment:
I meant to put this on the type's field, up where I commented :)

Reply:
Brain fart, I did know that 😅